pkg/domain/stateDiffGlobalConfig.go

```diff
@@ -0,0 +1,42 @@
+package domain
+
+import (
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/common"
+	"github.com/cloudogu/k8s-blueprint-operator/pkg/domain/ecosystem"
+)
+
+func determineGlobalConfigDiffs(
```

Comment:
🚫 **[checkstyle]** <unused> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>func `determineGlobalConfigDiffs` is unused